README.md

```diff
@@ -113,6 +113,40 @@ SLS_DEBUG = *
 BRANCH = local
 ```
 
+*GET* operations are exposed to the VTA App via API Gateway.
+```
+https://api.gateway.uri/develop/test-stations/P12345
+```
+*PUT* operations are via direct lambda invocation with an EventBridgeEvent object.
```

Comment:
Could you mention here that you actually have to use the post endpoint to mimic event bridge

Reply:
Done.